src/helper/helper.js

```diff
@@ -236,18 +235,18 @@ function buildAbstractModelInterface_(dataLayerHelper) {
  * If the method is a valid function of the value, the method will be applies
  * with any arguments passed in.
  *
- * @param {Array.<Object>} command The array containing the key with the
+ * @param {*} command The array containing the key with the
  *     method to execute and optional arguments for the method.
- * @param {Object|Array} model The current dataLayer model.
+ * @param {!Object<*>|!Array<*>} model The current dataLayer model.
  * @private
  */
 function processCommand_(command, model) {
```

Comment:
Shouldn't the first param be of !Array<*> type? I think getting to this point of the code is only possible if plain.type gives 'array', so you can safely cast it